src/ProgressOnderwijsUtils/Extensions/DictionaryExtensions.cs

```diff
@@ -48,11 +50,15 @@ public static T Field<T>([NotNull] this IReadOnlyDictionary<string, object> dict
         /// <param name="dict">The dictionary to extract  from</param>
         /// <param name="key">The key whose value to get.</param>
         /// <returns>The value of the key, or the default if the dictionary does not contain the key.</returns>
-        public static TValue GetOrDefaultR<TKey, TValue>([NotNull] this IReadOnlyDictionary<TKey, TValue> dict, [NotNull] TKey key)
-            => GetOrDefaultR(dict, key, default(TValue));
+        [return: MaybeNull]

```

Comment:
wat is de functie hiervan? Zit dit niet verwerkt in waar ie generic op is? Of betekent dit zoveel dat als ie generic is op TValue met Henk dat ik hier een 'Henk?' moet verwachten?

Reply:
dat laatste (hij kan null returnen ook al is T not nullable).